lib/cpp/antaris_sdk_environment.cc

```diff
@@ -107,9 +106,16 @@ static void parse_a_conf(char *conf_line, conf_t *out_conf)
 
 static void update_a_conf(char *conf_line)
 {
+    int8_t length = -1;
     conf_t a_conf = {0};
+    size_t length = 0;
```

Comment:
This is causing cmake to error when building our app:
```
120.1 [ 27%] Built target paapis_lib
120.4 [ 30%] Building CXX object _deps/antaris-sdk-build/CMakeFiles/antaris_api.dir/lib/cpp/antaris_sdk_environment.cc.o
120.9 [ 32%] Linking CXX static library ../../../lib/libsignals.a
121.3 /code/build/_deps/antaris-sdk-src/lib/cpp/antaris_sdk_environment.cc: In function 'void update_a_conf(char*)':
121.3 /code/build/_deps/antaris-sdk-src/lib/cpp/antaris_sdk_environment.cc:111:12: error: conflicting declaration 'size_t length'
121.3   111 |     size_t length = 0;
121.3       |            ^~~~~~
121.3 /code/build/_deps/antaris-sdk-src/lib/cpp/antaris_sdk_environment.cc:109:12: note: previous declaration as 'int8_t length'
121.3   109 |     int8_t length = -1;
121.3       |            ^~~~~~
121.5 make[2]: *** [_deps/antaris-sdk-build/CMakeFiles/antaris_api.dir/build.make:118: _deps/antaris-sdk-build/CMakeFiles/antaris_api.dir/lib/cpp/antaris_sdk_environment.cc.o] Error 1
```

https://github.com/antaris-inc/SatOS-Payload-SDK/pull/25 should fix it (testing that right now).